Mathlib/Analysis/NormedSpace/UnitizationL1.lean

```diff
@@ -0,0 +1,135 @@
+/-
+Copyright (c) 2024 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux
+-/
+import Mathlib.Algebra.Algebra.Unitization
+import Mathlib.Analysis.NormedSpace.ProdLp
+
+/-! # Unitization equipped with the $L^1$ norm
+
+In another file, the `Unitization 𝕜 A` of a non-unital normed `𝕜`-algebra `A` is equipped with the
+norm inherited as the pullback via a map (closely related to) the left-regular representation of the
+algebra on itself.
+
+However, this construction is only valid (and an isometry) when `A` is a `RegularNormedAlgebra`.
+Sometimes it is useful to consider the unitization of a non-unital algebra with the $L^1$ norm
+instead. This file provides that norm on the type synonym `WithLp 1 (Unitization 𝕜 A)`, along
+with the algebra isomomorphism between `Unitization 𝕜 A` and `WithLp 1 (Unitization 𝕜 A)`.
```

Comment:
Perhaps worth a note somewhere that `TrivSqZeroExt` has the same instance available without a type synonym. (and perhaps in the other direction talking about Unitization`)